src/tests/clipboard.test.tsx

```diff
@@ -214,7 +214,7 @@ describe("Paste bound text container", () => {
       await sleep(1);
       expect(h.elements.length).toEqual(2);
       const container = h.elements[0];
-      expect(container.height).toBe(354);
+      expect(container.height).toBe(368);
```

Comment:
don't know, but I believe the previous 354 number was incorrect. There were some bugs related to previously implementation which could be it. Or not.

(applies to the `770` assertion below as well)